gconf/gconf.go

```diff
@@ -5,9 +5,14 @@ import (
 	"github.com/iov-one/weave/errors"
 )
 
+// ReadStore is a subset of weave.ReadOnlyKVStore.
+type ReadStore interface {
```

Comment:
How about `StoreGetter`?

Naming this properly is hard. Following [effective Go recommendation](https://golang.org/doc/effective_go.html#interface-names) this interface should be `Getter`. `Getter` can mean absolutely anything and therefore is a terrible name.

Reply:
Well, Getter is a term used in a variety of languages for getting variables, looks horrible in an interface, but indeed complies with Go style. Just gotta live with that I guess. 

In this particular case the interface seems to be named proper, if we look at the parent expanded interface which reads `ReadOnlyKVStore`.